From de50ae35ac3fc6e8c2d195f232f7c43d8a7815f1 Mon Sep 17 00:00:00 2001
From: Julien Thomas <jthomas@zenetys.com>
Date: Sun, 16 Aug 2020 18:38:43 +0200
Subject: [PATCH] Fix redefinition warning of CAPTURE_ID_LEN and CAPTURE_FORMAT

This warning was introduced by commit 44ce937 "[fix] grok is not working
anymore with PCRE >= 8.34 ..." from Guillaume Bour
<gbour@orange-vallee.net>.
---
 grok.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/grok.h b/grok.h
index 509bfae..9e6397b 100644
--- a/grok.h
+++ b/grok.h
@@ -123,8 +123,8 @@ extern int g_cap_definition;
 /** grok_exec did not match your string */
 #define GROK_ERROR_NOMATCH 7
 
-#define CAPTURE_ID_LEN 4
-#define CAPTURE_FORMAT "%04x"
+#define CAPTURE_ID_LEN 5
+#define CAPTURE_FORMAT "_%04x"
 
 #include "grok_logging.h"
 
-- 
2.21.1

